src/ansys/fluent/core/session.py

```diff
@@ -435,7 +435,7 @@ def chdir(self, path: str) -> None:
         path : str
             Path of the directory to change.
         """
-        self._app_utilities.change_client_cortex_dir(path)
+        self._app_utilities.set_cortex_working_directory(path)
```

Comment:
@hpohekar The issue discusses simultaneously changing the client working directory (`os.chdir(...)`) but that would not make the change session-specific so our options appear limited there and I think we should take no additional action in that respect. I think that the user can just call `os.chdir(...)` independently. Perhaps we should document that in the docstring of this method.

Reply:
@seanpearsonuk `os.chdir()` does not change client working directory. I have already verified it. Only changing Cortex directory is not sufficient. The changed directory should be synchronized with the client as well.

Reply:
@hpohekar I don't follow what you mean when you say that `os.chdir(...)` does not change the client working directory. I think you mean something different to what you're saying but I don't know what that is. Maybe you mean that even though the working directory has changed, some other side effect that you had assumed would happen has not happened. Please could you help by clarifying.

Reply:
discussed offline.